src/pinns_pde_solve.jl

```diff
@@ -42,19 +42,21 @@ struct PhysicsInformedNN{isinplace,C,T,P,PH,DER,PE,AL,K} <: AbstractPINN{isinpla
         else
             initθ = init_params
         end
+        flat_initθ = if (typeof(chain) <: AbstractVector) vcat(initθ...) else  initθ end
```

Comment:
```suggestion
        flat_initθ = if (typeof(chain) <: AbstractVector) reduce(vcat,initθ) else  initθ end
```

Reply:
why is flattening necessary?

Reply:
in order to pass to `GalacticOptim.OptimizationProblem(f, flat_initθ)`